README.md

```diff
@@ -1,133 +1,111 @@
-# Ankermake M5 Protocol
+# AnkerMake M5 Protocol
 
-Welcome! This repository contains `ankerctl.py`, a command-line interface for
-monitoring, controlling and interfacing with Ankermake M5 3D printers.
+Welcome! This repository contains `ankerctl.py`, a command-line interface for monitoring, controlling and interfacing with AnkerMake M5 3D printers.
 
-NOTE: This software is in early stages, so expect sharp edges and occasional errors!
+**NOTE:** This software is in early stages, so expect sharp edges and occasional errors!
```

Comment:
Overall question: Should we still be saying this? It might be a little defeatist for version v1.0.0 :)

Reply:
Changing to: "**NOTE:** This is our first major release and while we have tested thoroughly there may be bugs. If you encounter one please open a [Github Issue](https://github.com/Ankermgmt/ankermake-m5-protocol/issues/new/choose)"